dev/com.ibm.ws.microprofile.config.1.4.cdi/src/com/ibm/ws/microprofile/config14/cdi/Config14PropertyLiteral.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017, 2019 IBM Corporation and others.
```

Comment:
What's with this change?

Reply:
We didn't start anything on Config 1.4 until 2019